README.md

```diff
@@ -8,7 +8,6 @@ Additional documentation for this sample can be found on [Microsoft Docs](https:
 
 - Start a new group call
 - Join an existing group call
-- Join an existing Teams Meeting (For instructions: [Teams Tenant Interoperability](https://docs.microsoft.com/azure/communication-services/concepts/teams-interop))
 - Render remote participant video streams
```

Comment:
@ddematheu2, here is the line I removed in the main branch. I'll let you know after the main and Teams_interop branches are all setup